enterprise/frontend/src/metabase-enterprise/tenants/index.tsx

```diff
@@ -138,6 +140,7 @@ export function initializePlugin() {
           <IndexRoute component={TenantsListingApp} />
           <Route path="" component={TenantsListingApp}>
             <ModalRoute path="new" modal={NewTenantModal} noWrap />
+            {PLUGIN_TENANTS.userStrategyRoute}
```

Comment:
This is so we can show the user strategy modal under the tenants page, using the appropriate `/admin/tenants/user-strategy` route url.

<img width="1984" height="1528" alt="CleanShot 2568-12-18 at 21 47 18@2x" src="https://github.com/user-attachments/assets/320e2438-484d-491b-bc10-b40ccc2a4830" />